src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -345,6 +346,10 @@ def get(self, request: Request, organization) -> Response:
         self.add_cursor_headers(request, response, cursor_result)
 
         # TODO(jess): add metrics that are similar to project endpoint here
+        transaction = Hub.current.scope.transaction
+        if transaction:
+            transaction.set_measurement("issues.unresolved", len(cursor_result))
```

Comment:
why don't we have a global `sentry_sdk.set_measurement` for this?

Reply:
commented above
> eventually we'll add top-level api helpers, but since this is experimental this is not exposed all the way yet.